resources/costing/ResourceFile_Consumables_Inflow_Outflow_Ratio.csv

```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
```

Comment:
should this file be on this PR, or the costing one?

Reply:
Good point, Tim. The script that generates this resource file (`consumable_availability_estimation.py`) is in under this PR (#1367) but it's used in the costing module and I had manually imported it there (which is probably not best practice). I can undo this commit in the costing PR #1637 and merge it in from here perhaps? The RF on this PR (PR #1367) is the final one and should override the one in costing.

Reply:
Ok -- if this is the final version of the file we want then I would propose the simplest thing to do would be :
* we leave this one here and we'll merge it imminently.
* then in the costing branch, when we merge in master, we make sure this version (which becomes the master version) over-writes the out-dated version on the costing branch.

Does that sound ok?